src/AI/FightClub.java

```diff
@@ -338,7 +338,7 @@ public GameResults runGame(GameInstance game, PrintStream defaultOut)
         stopwatches.put(game.activeArmy, stopwatches.get(game.activeArmy) + thinkTimeNanos);
 
         // Map should-ish be covered in units by turncount == map area
-        if(game.getCurrentTurn() > game.gameMap.mapWidth * game.gameMap.mapHeight)
+        if(game.getCurrentTurn() > Math.max(game.gameMap.mapWidth, game.gameMap.mapHeight))
```

Comment:
So like, Spann Island matches would have to be over in 15 turns? Seems short, especially for large, squarish maps.

Reply:
Just ran a Jakeman/Strong v Muriel/Patch on Spann that took 70 turns. Jakeman looked like he should have had it in the bag around 15, but then it swung back the other way (Muriel upgraded to Mechs for standard infantry and built Md/Neo tanks, but Jake stuck with inf/tanks).
And sure, Spann isn't exactly peak design for competitive maps, but that time limit seems overly restrictive if the goal is to see how things end.

Reply:
FWIW, it checks if there is an overwhelming value advantage instead of having a *strict* turn limit.
I guess we could raise it back up, but I'm not sure how much I care about the amount of noise there is since there's so much variance in the rest of the system.

Reply:
Ah, so this is just allowing an earlier end to the fight, gotcha. It could still skew results (and I think there's a good chance it would have in the fight I just ran), but that might still be more or less reasonable.